u_common.go

```diff
@@ -20,10 +20,11 @@ const (
 	utlsExtensionExtendedMasterSecret uint16 = 23 // https://tools.ietf.org/html/rfc7627
 
 	// extensions with 'fake' prefix break connection, if server echoes them back
-	fakeExtensionChannelID uint16 = 30032 // not IANA assigned
-
-	fakeCertCompressionAlgs uint16 = 0x001b
-	fakeRecordSizeLimit     uint16 = 0x001c
+	fakeExtensionTokenBinding uint16 = 24
+	fakeExtensionChannelIDOld uint16 = 30031 // not IANA assigned
+	fakeExtensionChannelID    uint16 = 30032 // not IANA assigned
```

Comment:
Will you implement the `fakeExtensionChannelIDOld` as well a la https://github.com/getlantern/utls/blob/9c5b596cbacc9beaf15ff2918930ddcd8401c60a/u_tls_extensions.go#L677-L697 ?

Reply:
Sure, will do.

Reply:
[How's this?](https://github.com/getlantern/utls/pull/9/commits/10e727c076d99a75e55bc7ef7aae71e7d148d630)

Current behavior is unchanged.  You can specify `FakeChannelIDExtension{oldExtensionID: true}` if you want to use the old extension ID.  I can't find any evidence that the structure of the extension in the ClientHello changed.  It seems like it's always been an empty structure used to signal client support for the extension.

On a related note, I realized that the fields in this struct and the `FakeTokenBindingExtension` I added need to be exported.  I'll fix that.

Reply:
Oh shoot I just realized I commented in that other line, but I think the only issue is that Sergey may want it to be backwards compatible API wise, so maybe we'll want to make it default to the previous behavior?

Reply:
It does default to the previous behavior.  It should be fully backwards compatible.

Reply:
To clarify, "old extension ID" refers to the extension ID that is older as far as TLS is concerned.  The existing extension ID in `utls` is the _new_ one and we're adding the old one.